causalml/feature_selection/filters.py

```diff
@@ -191,7 +198,12 @@ def _GetNodeSummary(data,
         for ti in treatment_group_keys: 
             results.update({ti: {}}) 
             for ci in y_name_keys:
-                results[ti].update({ci: results_series[ti, ci]}) 
+                if smooth:
+                    results[ti].update({ci: results[ti].update({ci: results_series[ti, ci]
```

Comment:
Seems `ci` will all be `None` as dictionary updating method doesn't return any value, I don't think that's what you are trying to do here?

Reply:
Fixed. That was a clumsy copy-paste job that I overlooked somehow.